testsuite/MDAnalysisTests/datafiles.py

```diff
@@ -501,6 +501,7 @@
 DLP_HISTORY = resource_filename(__name__, "data/dlpoly/HISTORY")
 DLP_HISTORY_order = resource_filename(__name__, "data/dlpoly/HISTORY_order")
 DLP_HISTORY_minimal = resource_filename(__name__, "data/dlpoly/HISTORY_minimal")
+DLP_HISTORY_minimal_cell = resource_filename(__name__, "data/dlpoly/HISTORY_minimal_cell")
```

Comment:
This also needs adding to the `__all__` list (see lines 39->209)

Reply:
Thanks, missed that!